<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>678</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/img/quizzyLogoTransparent.png</normaloff>:/resource/img/quizzyLogoTransparent.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">/*QWidget#centralWidget{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(173, 83, 137, 255), stop:1 rgba(60, 16, 83, 255));}*/

/* további lehetséges hátterek:
border-image: url(:/resource/img/bckg_5.jpg);
border-image: url(:/resource/img/bckg_7.jpg);
border-image: url(:/resource/img/bckg_10.jpg);
*/

QWidget#centralWidget
{
/*background-image: url(:/resource/img/backg_1.jpg);*/
border-image: url(:/resource/img/backg_1.jpg);
background-repeat: no-repeat, repeat;
bacground-position: center;
/*background-size: cover;
position: relative;*/
}

QPushButton:hover
{
   background-color:white;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>181</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left: 130px;
padding-right: 130px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/resource/img/loginTrans3.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usernameLogin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>70</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">/*
background-image: url(:/resource/img/kicsi_login_szurke.png);
background-repeat: no-repeat, repeat;
background-attachment: fixed;
background-position:  left;

padding-left: 30px;
text-align:justify; 
letter-spacing: 10px;
font: 14pt &quot;MS Shell Dlg 2&quot;;*/

QLineEdit {
background-image: url(:/resource/img/kicsi_user_login.png);
/*background-image: url(:/resource/img/kicsi_login_szurke.png);*/
background-repeat: no-repeat;
padding-left:10px;
background-attachment: fixed;
background-position:  left;

padding: 1px;
border-style: solid;
/*border: 2px solid gray;*/
border-radius: 8px;

padding-left: 30px;
text-align:justify; 
/* letter-spacing: 10px; /* valamiért nem akar működni ez */
font: 14pt &quot;MS Shell Dlg 2&quot;;
}
</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Felhasználónév</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordLogin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/*
background-image: url(:/resource/img/lock_icon.png);
background-repeat: no-repeat, repeat;
background-attachment: fixed;
background-position:  left;*/

/*
padding-left: 30px;
text-align:justify; 
letter-spacing: 10px;
font: 14pt &quot;MS Shell Dlg 2&quot;;*/

QLineEdit {
background-image: url(:/resource/img/lock_icon.png);
background-repeat: no-repeat;
background-attachment: fixed;
background-position:  left;

padding: 1px;
border-style: solid;
/*border: 2px solid gray;*/
border-radius: 8px;

padding-left: 30px;
text-align:justify; 
/* letter-spacing: 10px; /* valamiért nem akar működni ez */
font: 14pt &quot;MS Shell Dlg 2&quot;;
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Jelszó</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
color: white;
font: bold 14pt &quot;MS Shell Dlg 2&quot;;
letter-spacing: .4rem;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #da70d6, stop: 0.1 #ba55d3,  stop: 0.49 #8b008b );
border-radius: 7;
/*ccc*/
}
QPushButton:hover {
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #D7BDE2, stop: 0.1 #C39BD3,  stop: 0.49 #BB8FCE );
}
</string>
        </property>
        <property name="text">
         <string>LOGIN</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;

color: rgb(0, 255, 255);


</string>
        </property>
        <property name="text">
         <string>Forget password?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
